enterprise/apim/3.x/gateway/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-FROM openjdk:11-jre-slim-buster
+FROM eclipse-temurin:11-jre-focal
```

Comment:
Don't know if this is an impact of the base image change, but some docker images like this one don't build nomore.
Cause of the `apt-get upgrade`, asking confirmations to upgrade packages.
(see error example below).
Maybe we need to add a "-y" parameter ?

_...
The following packages will be upgraded:
  tzdata
1 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 295 kB of archives.
After this operation, 2,048 B disk space will be freed.
Do you want to continue? [Y/n] Abort.
The command '/bin/sh -c apt-get update     && apt-get upgrade     && apt-get --yes install wget unzip htop 	&& wget ${GRAVITEEIO_DOWNLOAD_URL}/graviteeio-ee-full-${GRAVITEEIO_VERSION}.zip --no-check-certificate -P /tmp     && unzip /tmp/graviteeio-ee-full-${GRAVITEEIO_VERSION}.zip -d /tmp/     && mv /tmp/graviteeio-full-${GRAVITEEIO_VERSION}/graviteeio-gateway* ${GRAVITEEIO_HOME}     && rm -rf /tmp/*     && rm -rf /var/lib/apt/lists/* 	&& chgrp -R 0 ${GRAVITEEIO_HOME}     && chmod -R g=u ${GRAVITEEIO_HOME}' returned a non-zero code: 1_

Reply:
I have updated the upgrade command, somehow missed that, should never context switch when working with Dockerfiles.